server/internal/app/config.go

```diff
@@ -68,6 +68,14 @@ type Config struct {
 
 	// server
 	Server ServerConfig `pp:",omitempty"`
+
+	// Health Check Configuration
+	HealthCheck HealthCheckConfig `pp:",omitempty"`
+}
+
+type HealthCheckConfig struct {
+	Username string `default:"" envconfig:"HEALTH_CHECK_USERNAME"`
+	Password string `default:"" envconfig:"HEALTH_CHECK_PASSWORD"`
```

Comment:
```suggestion
	Username string `pp:",omitempty"`
	Password string `pp:",omitempty"`
```
and add Password to the secrets so it will not printed in logs

Reply:
@yk-eukarya which secrets that you mean? Can you give some reference?

Reply:
@yk-eukarya if I use ``pp:",omitempty"`` how does it know which env variable to load?

Reply:
@yk-eukarya updated the .env example